R/getQuote.R

```diff
@@ -48,9 +55,10 @@ function(Symbols,what=standardQuote(),...) {
                 SymbolsString,
                 "&fields=",QFc)
   # The 'response' data.frame has fields in columns and symbols in rows
-  response <- jsonlite::fromJSON(URL)
+  response <- jsonlite::fromJSON(curl::curl(URL))
   if (is.null(response$quoteResponse$error)) {
     sq <- response$quoteResponse$result
+    Symbols <- sq$symbol
```

Comment:
This overwrites the `Symbols` function argument. That can confuse future you, if you don't remember that the object gets redefined somewhere in the function body.

Reply:
makes sense